types/react-navigation/index.d.ts

```diff
@@ -534,7 +534,7 @@ export function StackNavigator(
 ): NavigationContainer;
 
 // DrawerItems
-export const DrawerItems: React.ComponentClass<any>;
+export const DrawerItems: (props: any) => React.ReactElement<any> | React.ComponentClass<any>;
```

Comment:
Should here be better use `React.StatelessComponent<any>` ?

Reply:
Which will basicly equal to `React.ComponentType`

Reply:
ah cool I didn't know about that one 👍